databricks/koalas/tests/test_dataframe.py

```diff
@@ -69,6 +69,12 @@ def test_dataframe(self):
 
         self.assertEqual(ddf.a.notnull().alias("x").name, "x")
 
+        # check ks.DataFrame(ks.Series)
+        kser = pd.Series([1, 2, 3], name='x')
+        pser = ks.Series([1, 2, 3], name='x')
```

Comment:
`kser` and `pser` are upside down?

Reply:
I'll fix it right now.Excuse me. How to mention it in the doc?

Reply:
Maybe we should say like `If data is a Koalas Series, ...`.